packages/lab/stories/avatar.stories.newapp-avatar.css

```diff
@@ -0,0 +1,9 @@
+.uitk-light.uitk-newapp .uitkAvatar {
```

Comment:
These could be made look more different i guess? They look very similar to out of box ones.